src/py/aspen/config/config.py

```diff
@@ -244,5 +244,5 @@ def DATABASE_URI(self) -> str:
             instance_address = instance_info["Endpoint"]["Address"]
             instance_port = instance_info["Endpoint"]["Port"]
             instance_address = f"{instance_address}:{instance_port}"
-        db_name = os.getenv("REMOTE_DEV_PREFIX", "/aspen_db")
+        db_name = os.getenv("REMOTE_DEV_PREFIX") or "/aspen_db"
```

Comment:
This is a workaround to make sure we use "aspen_db" if the REMOTE_DEV_PREFIX env var is an empty string.